package.json

```diff
@@ -44,6 +44,7 @@
     "webpack-stream": "^3.1.0"
   },
   "devDependencies": {
+    "babel-plugin-transform-object-rest-spread": "^6.22.0",
```

Comment:
Not `stage-2`, but `stage-3`. I'm fine with using that instead if you think it's a good idea.

https://babeljs.io/docs/plugins/preset-stage-3/